base_layer/wallet_ffi/src/lib.rs

```diff
@@ -231,6 +236,49 @@ pub type TariComAndPubSignature = ComAndPubSignature;
 pub type TariUnblindedOutput = UnblindedOutput;
 pub struct TariUnblindedOutputs(Vec<UnblindedOutput>);
 
+/// PayRef Configuration FFI Types
```

Comment:
Can we move all of this to a separate file we did with `ffi_basenode_state`

Reply:
We can also add from method to convert from the tari core methods to these c structs